Testscripts/Windows/VALIDATE-INTEL-SGX-DRIVER.ps1

```diff
@@ -31,7 +31,7 @@ function Main {
         }
     }
 
-    if (($ubuntuVersion -notmatch "Ubuntu 18.04") -and ($ubuntuVersion -notmatch "Ubuntu 16.04")) {
+    if ($ubuntuVersion -notmatch "Ubuntu 18.04") {
```

Comment:
does it mean all sgx cases only can run on ubuntu 18.04?

Reply:
Yes, the latest version of OpenEnclave has abandoned 16.04 support.

Reply:
during the internal mail session, will it support higher ubuntu distro than 18.04 like ubuntu 20.04?